<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>250</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>550</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 255, 255);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="bt_abrir">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>50</y>
         <width>71</width>
         <height>70</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 217, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Abrir
 mesa</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_reservar">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>50</y>
         <width>75</width>
         <height>70</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(127, 127, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Reservar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_cancelar">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>50</y>
         <width>75</width>
         <height>70</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>75</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(255, 55, 55, 255), stop:1 rgba(255, 255, 255, 255));
font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
      <widget class="QLabel" name="lb_mesa">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>20</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 900 italic 11pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_agregar_productos">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>150</y>
         <width>85</width>
         <height>71</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>85</width>
         <height>80</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(0, 217, 0, 255), stop:1 rgba(255, 255, 255, 255));
font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Agregar
 Productos</string>
       </property>
      </widget>
      <widget class="QPushButton" name="bt_imprimir_cuenta">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>150</y>
         <width>80</width>
         <height>70</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>80</width>
         <height>70</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
font: 900 italic 12pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Imprimir
 Cuenta </string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
